### PR TITLE
Update AmiLoDService.php

### PR DESCRIPTION
**Pull request by alliomeria**
Add MeSH to LoD Reconciliation Service (form), to address https://github.com/esmero/ami/issues/185

### CODE REVIEW

File: src/AmiLoDService.php [hunk 1]
```diff
@@ -161,7 +161,13 @@ class AmiLoDService {
     'getty;aat;fuzzy' => 'Getty aat Fuzzy',
     'getty;aat;terms' => 'Getty aat Terms',
     'getty;aat;exact' => 'Getty aat Exact Label Match',
-    'wikidata;subjects;thing' => 'Wikidata Q Items'
+    'wikidata;subjects;thing' => 'Wikidata Q Items',
+    'mesh;term;startswith' => 'MeSH (starts with Term)',
+    'mesh;term;contains' => 'MeSH (contains Term)',
+    'mesh;term;exact' => 'MeSH (exact Term)',
+	'mesh;descriptor;startswith' => 'MeSH (starts with Descriptor)',
```
Comment (DiegoPino) [inline src/AmiLoDService.php:168]: @alliomeria could you please remove the extra spacing here? there are a few ones that are indented to the right. Thanks!!

Comment (alliomeria) [reply src/AmiLoDService.php]: Ok, all set @DiegoPino

**Review by DiegoPino — APPROVED** [review]
Awesome! Thanks @alliomeria. Merging!

**Comment by alliomeria** [comment]
Thanks to you @DiegoPino !

**Pull request merged**